packages/admin-ui-docs/docs/guidelines/components/select.mdx

```diff
@@ -105,3 +122,19 @@ Besides the props listed bellow, you can use all the props accepted by the `sele
 | tone         | `neutral, critical` | Select's tone of voice    | 🚫       | `neutral` |
 | criticalText | `string`            | TextArea critical message | 🚫       | -         |
 | csx          | `StyleProp`         | Defines component styles  | 🚫       | `{}`      |
+
+</Panel>
+<Panel id="designReference">
+
+## Do’s
+
+1. Mind the order of the options, like putting the more probable one to be selected at the top. In doubt, sort them alphanumerically (from A to Z and from 0 to 9).
+2. If there are more than four options listed, consider using a radio button.
```

Comment:
less than

Reply:
How so?

Reply:
instead of "more than" it should be "less than"

Reply:
Done!